src/lib/operators/join_reference_operator.cpp

```diff
@@ -0,0 +1,220 @@
+#include "join_reference_operator.hpp"
```

Comment:
(random line) @Bouncner @mrks What do you think of `JoinVerificationOperator` as a name? The term "verify" is used with similar semantic in the `--verify` option of the benchmarks, that's why I think it may be a good fit for something that provides a ground truth.

Reply:
Sounds good to me. I like the term especially since I don’t think somebody will use this operator accidentally. It’s clearer than reference.

Reply:
`JoinVerification` without "*Operator" suffix it is. Suffix dropped due to @mrks implicit dismissal https://github.com/hyrise/hyrise/pull/1577#discussion_r279195268